admin/includes/modules/copy_product_confirm.php

```diff
@@ -111,10 +111,15 @@
                                       FROM " . TABLE_PRODUCTS_DESCRIPTION . "
                                       WHERE products_id = " . $products_id);
         foreach ($descriptions as $description) {
+            $name = zen_db_input($description['products_name']); 
+            $newname = $name . " " . TEXT_DUPLICATE_IDENTIFIER; 
+            if (strlen($newname) > zen_field_length(TABLE_PRODUCTS_DESCRIPTION, 'products_name')) {
+               $newname = $name; 
```

Comment:
For feedback: Instead of setting `$newname = $name` (which leaves it less obvious which one is the duplicate), why not truncate at the length of the field?

Reply:
Because then you'd wind up with "Name of product CO" or something like that if COPY were half truncated.

Reply:
True. Putting "COPY" at the beginning would avoid that.